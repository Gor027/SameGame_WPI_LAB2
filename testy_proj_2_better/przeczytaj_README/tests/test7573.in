 N>NNN  N   NN >>N N>N  >N>N>>>>>N> >>> >N>>>NN NNN N  NNNN > >N
N >>NN>>>N > > 
>> NN >N    N  NNN>>N   > N>NN>>>N NNN>NN  N>>N >>> NNN> >  >>>>NN >>>
>NN >N > > >>  N>NN>>N N>N 
> NNN   N   > >>>N N>>N> N>N NN>N >N>N >  >N>    >   N
>NN  N >N> N
N  N>N>    >> >N>  >>>> >  >    >NNNN>N > 
 >>N > >N>N>>NN     >   NN  >>N    >NNN   >N >
>>>>N>NNNN N>>NN>  N 
    N>>  N>NN>NN
>>>>  > >
>NN>N N N > >NN>>   N>N >  N  >NNN NN   >N>NNN>  >> 
  NNNNN >   > >N N>  >   >>  >>>N> N>>NN>N> NN  N>>  N>> N>>    N 
 N    
N >N> > >>NN  N>N NN N >N N >NN N
N>>>>  >N  N NN N   N  >> >> NN> >N   NNNN  NN N
>  N>> >>N  NN NN N  N  N  N> N     N>N>>N> N
NN > >N N>>N N >N>> >>N>N > >> N>>>>  NNN   N>> N> NNN>   >NN>>  NN >>> >> NN N
>NN> >> N>NN >>>N N    >>>>>> N >N
>>>N>
