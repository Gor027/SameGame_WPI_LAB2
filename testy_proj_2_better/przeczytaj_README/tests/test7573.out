   N
   >
   NN   N
N >NN   >      >
>N>NNN  N      N>
>>NN>N> N  >N  NN       >
>NN>NN> NN >NN>>>       >     >
>NN>>N> N> N>>>N>   >N  >     > >
N>>>NNN>>> >>NNN>   NN  > >N >N >N  > > >   >
>>>NN>>NN> >>>NN> >N>>  N N>>NN>NN NN>> N   >N
>>>NN>>>>NNN>>N>>NN>>> N>N>NNN>>>N N>NNN> > >>   N
>NN>N>N>>N>N>NNNNN>>>> >NN>>>>>>NN>NNNN>>N>>N>   >
NNNN>NN>N>>NNN>>NNNN>>N>>NNN>>NNN>>>N>NN>N>NNNN N>N  N
N>>>>>>N>NNN>NNNNNN>NN>>NN>>>NN>N>NNNNNNNN>NNNN >>> NNNN NN > >
>N>>>>N>N>>N>NN>NN>NNNN>N>>N>NN>>>NN>>N>>N>NNN> N>> NN>> >>>>>>NNN  >>
NNNN>>>>N>NNN>>>N>NN>>N>>>>>>>NN>N>>N>NNNN>NN>>NN>>NNN>>NN>NN>>>NNN>>>>>>NNN
